pkg/lightning/config/const.go

```diff
@@ -31,7 +31,7 @@ const (
 	// 	https://github.com/tikv/tikv/blob/e030a0aae9622f3774df89c62f21b2171a72a69e/etc/config-template.toml#L360
 	// lower the max-key-count to avoid tikv trigger region auto split
 	SplitRegionSize         ByteSize = 96 * units.MiB
-	SplitRegionKeys         int      = 1_280_000
+	SplitRegionKeys         int      = 960_000
```

Comment:
this is the default for TiKV, see https://github.com/tikv/tikv/blob/e030a0aae9622f3774df89c62f21b2171a72a69e/etc/config-template.toml#L361